src/mlpack/tests/cv_test.cpp

```diff
@@ -94,7 +94,31 @@ TEST_CASE("ConfusionMatrixTest", "[CVTest]")
   REQUIRE(output(1, 0) == 2);
   REQUIRE(output(1, 1) == 3);
 }
+/**
+ * Test for confusion matrix which outputs percentages.
+ */
+TEST_CASE("ConfusionMatrixPercentageTest", "[CVTest]")
+{
+  // Labels that will be considered as "ground truth".
+  arma::Row<size_t> labels("0 0 1 0 0  1 0 1 0 1");
 
+  // Predicted labels.
+  arma::Row<size_t> predictedLabels("0 0 0 0 0  1 1 1 1 1");
+  // Confusion matrix storing percentages
+  arma::Mat<double> output;
+  // numClasses argument has been set to three to test divide by zero error.
+  data::ConfusionMatrixPercentage(predictedLabels, labels, output, 3);
+  double result00 = 400.0/6.0;
+  double result01 = 100.0/4.0;
+  double result10 = 200.0/6.0;
+  double result11 = 300.0/4.0;
```

Comment:
```suggestion
  double result00 = 400.0 / 6.0;
  double result01 = 100.0 / 4.0;
  double result10 = 200.0 / 6.0;
  double result11 = 300.0 / 4.0;
```
Just some simple style fixes. :)